serverless.yml

```diff
@@ -178,8 +178,21 @@ stepFunctions:
                       Next: defaultMapFail
                   Next: loadToObservationDb
                 loadToObservationDb:
-                  Type: Pass
+                  Type: Task
+                  Resource: arn:aws:lambda:${self:provider.region}:${self:custom.accountNumber}:function:aqts-capture-ts-loader-${self:provider.stage}-loadTimeSeries
+                  ResultPath: $.timeSeries
+                  TimeoutSeconds: 60
                   End: true
+                  Retry:
+                    - ErrorEquals:
+                      - States.TaskFailed
+                      IntervalSeconds: 35
+                      MaxAttempts: 2
+                      BackoffRate: 2
+                  Catch:
```

Comment:
Drop the error handler. If this fails, we're just restart the entire map state.

Reply:
dropped